crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -125,6 +125,14 @@ where
         // anyone pushing onto this, because the handles will be unset
         &mut self.children
     }
+
+    pub fn len(&self) -> usize {
```

Comment:
What do you think of the name `len`? - my first impression would be that it tells you how many children there are. I might go for `num_codeunits` or `len_codeunits` or even `codeunits` - what do you think?

Reply:
I actually changed it to `text_len`. Maybe `len_codeunits` is a better name 🤔 .

Reply:
Happy with either